synquacer.mk

```diff
@@ -0,0 +1,127 @@
+################################################################################
+# Following variables defines how the NS_USER (Non Secure User - Client
```

Comment:
s/defines/define/
s/Non Secure/Non-secure/ (here and line below)

Reply:
edit pushed.